### PR TITLE
Added entry for event log ingestion issues

### PR DESCRIPTION
**Pull request by ms-smithch**
Included section for troubleshooting event log issues due to a trend in support cases.

### CODE REVIEW

**Comment by PRMerger15** [comment]
@ms-smithch : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: ATPDocs/troubleshooting-known-issues.md [hunk 1]
```diff
@@ -207,6 +207,27 @@ This is an engineering limitation.
...
+
+**Cause:**
+
+A Discretionary Access Control List is limiting access to the required event logs by the Local Service account.
+
+**Resolution:**
+
+Ensure that the Discretionary Access Control List includes the following entry:
+
+`(A;;0x1;;;S-1-5-19)`
```
Comment (thesourcerer) [inline ATPDocs/troubleshooting-known-issues.md:229]: The Entry should be 
(A;;0x1;;;S-1-5-80-818380073-2995186456-1411405591-3990468014-3617507088)

File: ATPDocs/troubleshooting-known-issues.md [hunk 2]
```diff
@@ -207,6 +207,27 @@ This is an engineering limitation.
 
 No known resolution.
 
+## Sensor fails to enumerate event logs
+
+If you observe a limited number or lack of security event alerts within the [!INCLUDE [Product short](includes/product-short.md)] console but no health alerts are triggered.
```
Comment (thesourcerer) [inline ATPDocs/troubleshooting-known-issues.md:212]: security event alerts or logical activities

**Comment by PRMerger12** [comment]
@shsagir : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@shsagir : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**